docs/source/working_with_workspaces.rst

```diff
@@ -57,9 +57,48 @@ would result in multiple workspaces being created with same name.
 Workspace sharing
 *****************
 
-A workspace in Modelon Impact can be shared as a zipped artifact(snapshot) or as workspace definition referring to version controlled 
+A workspace in Modelon Impact can be shared as a zipped artifact (snapshot) or as a workspace configuration.
+A snapshot can be either stored in a cloud storage and shared with the other users of the same 
+Modelon Impact site or downloaded as a zip file to the client system.
```

Comment:
Modelon Impact site - A bit ambiguous on what site means here. Group or website. Not sure if adding a section about tenants/groups and then use Modelon Impact group makes sence?